git_creation_and_tracking/git_creation_and_tracking.md

```diff
@@ -0,0 +1,780 @@
+Courtesy NASA/JPL-Caltech.
+[Mummy](https://commons.wikimedia.org/wiki/File:Mummy_icon_-_Noun_Project_4070.svg)
+$\copyright$; Gilad Fried / [The Noun Project](https://thenounproject.com/) /
+[CC BY 3.0](https://creativecommons.org/licenses/by/3.0/deed.en).
+[Moon](https://commons.wikimedia.org/wiki/File:Lune_ico.png)
+$\copyright$; Luc Viatour / [https://lucnix.be](https://lucnix.be/) /
+[CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/deed.en).
+
+In the examples, we will see code as if it is written by Vlad Dracula. If you are practicing these steps as you move through this module, your name and email address will appear instead of Dracula's.
+
+To copy the code in the examples, enter everything after the dollar sign "$" into the command line, and then press the enter or return key. If the command has any output, it will not start with a dollar sign, but a new line where you can type after a dollar sign will appear.
+
+```console
+$logname
+vlad
+$
+```
+
+The **codeblock** above tells you that when Dracula types `logname` and presses enter, the output is 'vlad', his username. Try entering `logname` in your own console. There will be other code in front of each `$` describing who and where you are. Since the stuff before the `$` will be different for everyone, it is customary to omit it from example code. We will also omit the `$` after the output.
+
+### Initializing the repository
+
+First, let's create a directory in `Desktop` folder for our work and then move into that directory:
+
+
+```console
+$ cd ~/Desktop
+$ mkdir planets
+$ cd planets
+```
+The `cd` command means **change directory**, and `mkdir` tells the computer to **make** a **directory** with the name "planets." Once we have a `planets` directory, we use `cd` open that directory.
+
+
+Next we tell Git to make `planets` a [repository](https://swcarpentry.github.io/git-novice/reference.html#repository) (also called a **repo**)
+-- a place where Git can store versions of our files:
+
+
+```console
+$ git init
+```
+
+
+It is important to note that `git init` will create a repository that
+includes subdirectories and their files---there is no need to create
+separate repositories nested within the `planets` repository, whether
+subdirectories are present from the beginning or added later. Also, note
+that the creation of the `planets` directory and its initialization as a
+repository are completely separate processes.
+
+If we use `ls`, the **list** command, to show the directory's contents,
+it appears that nothing has changed:
+
+```console
+$ ls
+```
+
+
+Like many commands, `ls` can be modified using **flags**, very short bits of code that change some of the options on the command. If we add the `-a` flag to `ls`, we will be asking Git to list all files.
+Now we can see that Git has created a hidden directory within `planets` called `.git`:
+
+```console
+$ ls -a
+
+.	..	.git
```

Comment:
what do the dots . and .. mean here?  Consider explaining

Reply:
good call, added a "learnmore" box!